[RHACS] Release notes and other updates for 3.69 (#43081)
openshift/openshift-docs: modules/secured-cluster-services-config.adoc

| `auditLogs.disableCollection`
| If you set this option to `true`, {product-title} disables the audit log detection features used to detect access and modifications to configuration maps and secrets.

| `scanner.disable`
| If you set this option to `false`, {product-title} deploys a lightweight scanner and Scanner DB in the secured cluster to allow scanning images on OpenShift Container Registry. Enabling Scanner is only supported on OpenShift. Defaults to `true`

| `scanner.dbTolerations`

> | If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scannerdb.

Reviewer: ```suggestion
| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner DB.
```

| `scanner.replicas`
| Resource specification for Collector's Compliance container.

| `scanner.logLevel`
| Setting this parameter allows you to modify the scanner log level. This should only be used for troubleshooting purposes.

| `scanner.autoscaling.disable`
| If you set this option to `true`, {product-title} disables autoscaling on the Scanner deployment.

| `scanner.autoscaling.minReplicas`
| The minimum number of replicas for autoscaling. Defaults to 2.

| `scanner.autoscaling.maxReplicas`
| The maximum number of replicas for autoscaling. Defaults to 5.

| `scanner.nodeSelector`
| Specify a node selector label as `label-key: label-value` to force Scanner to only schedule on nodes with the specified label.

| `scanner.tolerations`
| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner.

| `scanner.dbNodeSelector`
| Specify a node selector label as `label-key: label-value` to force Scanner DB to only schedule on nodes with the specified label.

| `scanner.dbTolerations`

> | If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scannerdb.

Reviewer: ```suggestion
| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner DB.
```

| `scanner.resources.requests.memory`
| The memory request for the scanner container. Use this parameter to override the default value.

| `scanner.resources.requests.cpu`

> | The CPU request for the scanner container. Use this parameter to override the default value.

Reviewer: ```suggestion
| The CPU request for the Scanner container. Use this parameter to override the default value.
```

| `scanner.resources.limits.memory`

> | The memory limit for the scanner container. Use this parameter to override the default value.

Reviewer: ```suggestion
| The memory limit for the Scanner container. Use this parameter to override the default value.
```

| `scanner.resources.limits.cpu`

> | The CPU limit for the scanner container. Use this parameter to override the default value.

Reviewer: ```suggestion
| The CPU limit for the Scanner container. Use this parameter to override the default value.
```
